corehq/ex-submodules/pillowtop/processors/elastic.py

```diff
@@ -68,20 +70,23 @@ def process_change(self, change):
 
         if change.deleted and change.id:
             doc = change.get_document()
+            domain = doc.get('domain') if doc else None
+            if not domain:
+                domain = change.metadata.domain
```

Comment:
Is there a chance this won't be present either?  I think you said that happens when deleting a full domain, is that right?

Reply:
I see there's conversation about this elsewhere - will move to that thread.